src/js/helpers.js

```diff
@@ -255,11 +250,6 @@ export function stringifyFormReplacer(key, value) {
       return undefined;
     }
 
-    // autosuggest widgets save value and label info, but we should just return the value
-    if (value.widget === 'autosuggest') {
-      return value.id;
-    }
-
     // Exclude file data
```

Comment:
I actually had questions about whether to remove this as well. I wasn't sure exactly how the uploading works. The data isn't attached to the form so how does the submitted form get associated with the uploaded file when the backend wants to do something with them both?

Reply:
I don't think there's uploading here? This is for an autosuggest field, not file upload. I assume it's just data being populated?

Reply:
I meant the block under the `// Exclude file data` comment that follows it. This is another duck-typed modification but I think it may be making vets.gov assumptions that don't apply generally.